src/components/Toaster/ToasterComponent/ToasterPortal.tsx

```diff
@@ -31,15 +31,13 @@ export function ToasterPortal({children, className, mobile}: Props) {
         };
     }, []);
 
-    React.useEffect(() => {
-        if (!el.current) {
-            return;
-        }
-
-        el.current.className = b({mobile}, className);
-    }, [className, mobile]);
-
-    return <Portal container={el.current}>{children}</Portal>;
+    return (
+        <Portal container={el.current}>
+            <div className={b({mobile}, className)} aria-live="assertive">
```

Comment:
floating ui render it's own `div`, why we add our own div?

Reply:
What div are your referring to? You mean we don't need the effect above?